app/views/admin/performance/support_queries/show.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= render partial: "admin/performance/nav" %>
```

Comment:
We don't currently consistent set the page title for all the admin pages, but let's try and do it for all the new ones... 😂 🤦 

```suggestion
<% content_for :title, "Support form stats" %>

<%= render partial: "admin/performance/nav" %>
```